apps/core/src/utils/determineCountDownText.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
shouldn't this header look like this?
```
// Copyright (c) 2024 IOTA Stiftung
// SPDX-License-Identifier: Apache-2.0
```

Reply:
Yes it's a new file, but the code is modified Mysten code from another file so I think its ok keeping the modification headers

Reply:
ah, got it! all good then :rocket: